sherpa/sim/sample.py

```diff
@@ -255,29 +279,39 @@ def get_scales(self, fit, myscales=None):
         scales : numpy array
             Two-dimensional scales array (npar by npar elements,
             matching the free parameters in fit). The values are the
-            covariance matrix for the parameters (or the input values if
-            given).
+            covariance matrix for the parameters (or the input values
+            if given).
```

Comment:
Not true any longer. The "values of the covariance matrix" is a mathematically well defined term and does not include scaling by sigma. We have to make a choice here what we actually want: Do we want to return the covariance or do we want to return scales (calculated from the covariance and sigma)? Given the name of the function, I think the latter is appropriate, but then the docstring here should match that.

Reply:
At some level this gets to my general uncertainty about this: is the `sigma` parameter really meaningful? For those cases where it ends up coming from the covariance matrix (or projection down to 1D) then it feels rather reductive, but there is an argument that it is meaningful when calculated from the data (but it's been long enough for me to forget what exactly that particular code path is doing).

However, we do have at least one user of this functionality (the CSC code), and in other places we have a `sigma` parameter.

This is part of what I want to talk to @anetasie about.

Reply:
If the CSC uses it, let's have it. All I'm asking then is to put some words to that effect into the docstring so that it doesn't just say "the values are the covariance matrix" when that's true only for the (default) `sigma=1`.